packages/enterprise-search/docs/index.asciidoc

```diff
@@ -0,0 +1,17 @@
+= enterprise-search-js
+
+:doctype:           book
+
+include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
+
+include::overview.asciidoc[]
+
+include::installation.asciidoc[]
+
+include::app-search-api.asciidoc[]
+
+include::workplace-search-api.asciidoc[]
+
+include::enterprise-search-api.asciidoc[]
+
+include::release-notes/index.asciidoc[]
```

Comment:
I suggest flattening this out (remove directory) to follow the pattern in the main Enterprise Search docs.